src/resources/handle-pull-request.js

```diff
@@ -24,7 +26,7 @@ module.exports = ({ emitter, config }) => (repository, callback) => {
     head: `${config.github.owner}:${config.github.branch}`,
     title: pullRequestMetadata.title,
     base: repository.manifestContent.workingBranch,
-    body: pullRequestMetadata.body
+    body: pullRequestMetadata.body.length > MAX_ALLOWED_BODY_CHARS ? "This repository has too many translation files, Mercury can't return a status description with the PR." : pullRequestMetadata.body.length
```

Comment:
Shouldn't the body be returned instead of length in the else clause?

```suggestion
    body: pullRequestMetadata.body.length > MAX_ALLOWED_BODY_CHARS ? "This repository has too many translation files, Mercury can't return a status description with the PR." : pullRequestMetadata.body
```

Reply:
Whoops! Amended